### PR TITLE
Community Resources and CoC

### PR DESCRIPTION
**Pull request by YazzyYaz**
Code of Conduct, community calendar and community section.

### CODE REVIEW

File: docs/community/calendar.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Community Calendar
+
+The Celestia Community Calendar is available for finding all the different community call events happening in Celestia's community.
+
+You can add the Community Calendar to your personal calendar [here](https://calendar.google.com/calendar/u/0?cid=Y19za2JzbjIzNWszYmlzdHNoZ3RvNmw5ODYyNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
```
Comment (liamsi) [inline docs/community/calendar.md:5]: Is this an existing calendar or was this calendar created for this speciically?

Comment (YazzyYaz) [reply docs/community/calendar.md:5]: It was created specifically for community events. I created it like 2-3 weeks ago

**Review by liamsi — APPROVED** [review]

File: docs/community/coc.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
```
Comment (adlerjohn) [inline docs/community/coc.md:32]: 🤔
```suggestion
* Insulting or derogatory comments, and personal or political attacks
```

Comment (liamsi) [reply docs/community/coc.md:32]: I'm ok to keep trolling out of our GitHub org.

**Pull request merged**